src/plugins/embeddable/public/lib/actions/edit_panel_action.ts

```diff
@@ -109,8 +110,17 @@ export class EditPanelAction implements Action<ActionContext> {
     const app = embeddable ? embeddable.getOutput().editApp : undefined;
     const path = embeddable ? embeddable.getOutput().editPath : undefined;
     if (app && path) {
-      const state = this.currentAppId ? { originatingApp: this.currentAppId } : undefined;
-      return { app, path, state };
+      if (this.currentAppId) {
```

Comment:
Whoever merges first will need to resolve this exact lines of codes in the other PR :)